presto-spi/src/main/java/com/facebook/presto/spi/PartitioningFilterCoverage.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi;
+
+/**
+ * Result indicating whether a set of filters on partition columns retricts the range of values
+ * on the columns
+ */
+public enum PartitioningFilterCoverage
```

Comment:
Note Hive partition is not a "partitioning" at engine level. Presto engine "partitioning" corresponds to Hive bucket.  Hive partition filter is modeled as table layout selection in Presto today :(

What about `TableLayoutFilterCoverage`?  cc @rschlussel

Reply:
Yes, I prefer TableLayoutFilterCoverage